ElectronClient/app/gui/SideBar.jsx

```diff
@@ -286,6 +286,21 @@ class SideBarComponent extends React.Component {
 			item = BaseModel.byId(this.props.folders, itemId);
 		}
 
+		if (itemType === BaseModel.TYPE_FOLDER && !item.encryption_applied) {
+			menu.append(
+				new MenuItem({
+					label: _('New sub-notebook'),
+					click: async () => {
```

Comment:
`async` can be removed here as there's no async calls.

Reply:
hmm, there are async calls in the function that is called. But I can try to remove it.

Reply:
As long as there's no `await` calls inside the function, `async` is not necessary.

Reply:
There's an await (`folder = await Folder.save({ title: answer }, { userSideValidation: true });`), but apparently it is not needed. In other areas an async is needed whenever there is one `await` in the callee chain.

This is weird though. In that case we can remove it from all other calls below as well. I only used them as a template. But ok, I removed it.